### ISSUE TITLE
QUESTION: Wordpress latest blog posts not showing

### ISSUE THREAD

**Issue by RaptureForums** [10up/ElasticPress#2272]
**Describe your question**

Hi,
I updated my ElasticPress Wordpress plugin around July 8th. After doing so my latest posts show on my Wordpress homepage as they are supposed to. However, in my blog listings section the latest posts showing are from July 8th when I updated the plugin. The last week worth of articles is not showing up on the blog article listings page.

I have re-done the Sync on my posts in the ElasticPress admin area and that does not help. 

Do you have ideas what maybe causing this? I have upgraded today to the latest version and it still happens.

Thank you for your help.

**Comment by felipeelia** [comment]
Hi @RaptureForums ,

Would it be possible for you to index via WP-CLI using `wp elasticpress index --setup --show-errors`? If so, can you please do that and share any output it returns to you? Thanks!

**Comment by RaptureForums** [comment]
Hi Felipe,
I'm sorry but I am not really a server person or very knowledgeable about technical things. Is WP-CLI something I will need to have installed on my server? If so, I will need to have my host install that on my server. 

Thank you for your help and patience with my question. :-)

**Comment by felipeelia** [comment]
Hi @RaptureForums!

Sorry, yes, WP-CLI is the "Command Line Interface" for WordPress, so with that, you can run a couple of commands in your server terminal and change your WordPress site.

Another way to test if your new content is being indexed is by installing [Debug Bar](https://wordpress.org/plugins/debug-bar/) and [Debug Bar ElasticPress](https://wordpress.org/plugins/debug-bar-elasticpress) plugins, searching for the post title and checking in Debug Bar ElasticPress results if (1) Elasticsearch is working and (2) it returned the post correctly.

**Comment by RaptureForums** [comment]
Hi Felipe,
Well, I installed those 2 plugins. And I then reactivated ElasticPress. I just saw it was recommended to run a full sync again so I did that. And for good measure I decided to purge my LiteSpeed and CloudFlare caches and it looks like it is working properly now. :-)

I think I ran the sync last time, but it's been a week and my memory is a little fuzzy so I am not sure if I did or not. 

Anyway, thank you for your help and I will monitor this to make sure I don't have any more "hiccups" going forward.

Take care. :-)
